core/controllers/suggestion.py

```diff
@@ -112,39 +112,48 @@ def post(self):
             self.render_json(self.values)
             return
 
-        new_image_filenames = (
```

Comment:
Will these changes fix #11904? I thought the image loading issue was occurring for existing suggestions that were using the old image storage paths. Aren't these changes for when we create new suggestions?

Reply:
I've removed these changes for now.